pkg/roachprod/opentelemetry/cockroachdb_metrics.go

```diff
@@ -415,16 +417,60 @@ var cockroachdbMetrics = map[string]string{
 	"changefeed_sink_batch_hist_nanos_bucket":                             "changefeed.sink_batch_hist_nanos.bucket",
 	"changefeed_sink_batch_hist_nanos_count":                              "changefeed.sink_batch_hist_nanos.count",
 	"changefeed_sink_batch_hist_nanos_sum":                                "changefeed.sink_batch_hist_nanos.sum",
+	"changefeed_sink_backpressure_nanos":                                  "changefeed.sink_backpressure_nanos",
+	"changefeed_sink_backpressure_nanos_bucket":                           "changefeed.sink_backpressure_nanos.bucket",
+	"changefeed_sink_backpressure_nanos_count":                            "changefeed.sink_backpressure_nanos.count",
+	"changefeed_sink_backpressure_nanos_sum":                              "changefeed.sink_backpressure_nanos.sum",
 	"changefeed_sink_io_inflight":                                         "changefeed.sink_io_inflight",
 	"changefeed_size_based_flushes":                                       "changefeed.size_based_flushes",
 	"changefeed_stage_checkpoint_job_progress_latency":                    "changefeed.stage.checkpoint_job_progress.latency",
+	"changefeed_stage_checkpoint_job_progress_latency_bucket":             "changefeed.stage.checkpoint_job_progress.latency.bucket",
```

Comment:
First commit is in #154487, I separated it out so that one can be backported further back